baus.py

```diff
@@ -105,6 +105,7 @@ def run_models(MODE):
         def get_baseyear_models():
 
             baseyear_models = [
+                #"test_namespace",
```

Comment:
Was this only used for the PR; should we remove?

Reply:
Ya that doesn't need to exist anymore. Removing.